src/lib.rs

```diff
@@ -845,6 +845,16 @@ impl Default for Ini {
     }
 }
 
+impl ToString for Ini {
+    fn to_string(&self) -> String {
+        let mut writer = String::new();
+        if let Err(err) = self.write_to_fmt(&mut writer) {
+            panic!("{}", err.to_string());
+        }
+        writer
+    }
+}
```

Comment:
Honestly this is negotiable, it could be nice to have but I'm perfectly fine cutting it out.